ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Nagios_Plugins/tasks/additional_plugins/check_label

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Nagios Plugin to check the percentage of machines online for a given label on ci.adoptopenjdk.net
+
+if [ -z $1 ] || [ -z $2 ] || [ -z $3 ]; then
+  echo "UNKNOWN - Invalid arguments"
+  echo "Usage: $0 <Label> <Warning_Level> <Critical_Level>"
+  exit 3
+fi
+
+if ! command -v jq &> /dev/null; then
+  echo "UNKNOWN - JQ isn't installed"
+  exit 3
+fi
+
+# Get list of machines in label
+mapfile -t machine_array < <(curl -s https://ci.adoptopenjdk.net/label/$1/api/json | jq '.nodes[] | .nodeName' | sed 's/\"//g') 
```

Comment:
What's the sed for? Chopping `"` from the JSON values? If so, the `-r` flag of jq might be helpful.